packages/field/src/FormControlMixin.js

```diff
@@ -76,7 +76,31 @@ export const FormControlMixin = dedupeMixin(
       }
 
       get inputElement() {
-        return (this.$$slot && this.$$slot('input')) || this.querySelector('[slot=input]'); // eslint-disable-line
+        if (!this.__inputElementCache) {
```

Comment:
improved version of it can be found at https://github.com/ing-bank/lion/pull/158

we should sync this code with that one...